script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1662,37 +1664,37 @@ For now, we should head toward the Church, right?"
 [sha:a8ff2029ae39eb1945a0daf272189187b13b0cff]{
 -- Page 496, Offset 34715. Glued. Mods: @x.
 -- でも。その前に、もっと大きな問題がありました。ロアなんかより、もっと大きな問題が」
-However. Before that, there's an even bigger problem. A problem that's bigger than Roa."
+However, there's another problem that we have to solve before that――an even bigger problem than Roa."
 }
 [sha:3fb7d0c71814bdec64e19b5206a57d36b58c2540]{
 -- Page 497, Offset 34716.
 -- 「？　ロアより大きな問題、ですか……？」
-"A problem bigger than Roa...?"
+"An even bigger problem than Roa...?"
 }
 [sha:68af02e113d29fb80ceaf700fac10a8eff8c4049]{
 -- Page 498, Offset 34717.
 -- 「……いえ、何でもありません。
-"...No, it's nothing to be concerned about.
+"...No, don't worry about it.
 }
 [sha:537748ccf6eb0d67accc57e6d8b25b32a095d473]{
 -- Page 498, Offset 34718.
 -- 今はここから出ましょう。わたしのアパートなら、少しは遠野くんを楽にしてあげられると思います」
-We should get out of here now. I think you'll be a little bit at ease if we go to my apartment."
+We should get out of here. I think you'll feel a little bit better if we go to my apartment."
```

Comment:
i know where this is going! I've read arc route!

Reply:
unfortunately not...............